books/free-programming-books-bn.md

```diff
@@ -64,6 +64,11 @@
 * [SL3 Framework - Code For Brain](https://web.archive.org/web/20201024204437/https://sl3.app) - Stack Learners *(:card_file_box: archived)*
 
 
+### Problem SOlving
+
+* [৫২ টি প্রোগ্রামিং সমস্যা](http://cpbook.subeen.com/p/blog-page_11.html) - Tamim Shahriar Subeen (HTML)
```

Comment:
This is blog page 11? Is there a link for the whole work?

Reply:
as per the link, the page might be called the blog page 11 but it is indeed the only page that contains everything from the book "52 Programming Problems".